utils/docker/Dockerfile.leap.15

```diff
@@ -177,7 +180,8 @@ ARG DAOS_DEPS_BUILD=yes
 # ensure that latest dependencies are used.
 USER root:root
 RUN [ "$DAOS_DEPS_BUILD" != "yes" ] || \
-    { zypper --non-interactive --no-gpg-checks update && zypper clean --all; }
+    { zypper --non-interactive --gpg-auto-import-keys --no-gpg-checks ref && \
+      zypper --non-interactive update && zypper clean --all; }
```

Comment:
I have a slight preference for each command to be on it's own line as it's easier to parse.  You have almost identical code on L196 and that's much clearer.

Reply:
Changed in next push.